.dictionary.txt

```diff
@@ -1,5 +1,3 @@
-!preform
```

Comment:
Commit message: mention that you removed the v5 hooks since that is what makes this change of type "feat" rather than "ci."